formatTest/unit_tests/expected_output/infix.re

```diff
@@ -891,30 +891,30 @@ let containingObject = {
     /* Same as */
     let result = 2 + (- add 4 0);
     /* That same example but with ppx attributes on the add application */
-    let result = 2 + (- add 4 0 [@ppx]);
+    let result = 2 + (- [@ppx] add 4 0);
     /* Same as */
-    let result = 2 + (- add) 4 0 [@ppx];
+    let result = [@ppx] 2 + (- add) 4 0;
```

Comment:
Maybe in this PR, the original *input* file might have been changed to different semantics as well, which explains why the output would be changed semantically.